packages/@webex/internal-plugin-calendar/src/calendar.js

```diff
@@ -294,7 +305,7 @@ const Calendar = WebexPlugin.extend({
         meetingObjects.forEach((meeting) => {
           if (!meeting.encryptedParticipants) {
             promises.push(
-              this.getParticipants(meeting.id).then((notesResponse) => {
+              this.getParticipants(meeting.participantsUrl).then((notesResponse) => {
```

Comment:
is it guaranteed that we always have `meeting.participantsUrl` at the point in time when this code is executed?